app/views/users/show.html.erb

```diff
@@ -30,7 +33,7 @@
     </div>
     <div class="col-md-6 col-lg-6" >
       <h3 class ="text-center" id = "supply-hist"> previously ordered supplies:</h3>
-      
+
```

Comment:
@kpearson I reviewed this pull request with my mentor @mikegehard and he gave me some good tips on how to keep commits small. One of them is to make whitespace commits separate from code change commits. Let me know if you'd like to hear more.